replica/distributed_loader.cc

```diff
@@ -726,6 +726,9 @@ future<> distributed_loader::populate_keyspace(distributed<replica::database>& d
     co_await coroutine::parallel_for_each(ks.metadata()->cf_meta_data() | boost::adaptors::map_values, [&] (schema_ptr s) -> future<> {
         auto uuid = s->id();
         lw_shared_ptr<replica::column_family> cf = column_families[uuid];
+        if (cf->is_ready_for_writes()) {
```

Comment:
The commit message didn't explain this particular change. Why is it needed? What happens to the `cf`s that we skip? The code below is creating directories for them, wouldn't that mean that these `cf`s won't have directories? Why is important for them to be "ready for writes" when we create directories etc.?

A comment here would be best, but it should also be explained in the commit message.